docs/devguide/naming_conventions.rst

```diff
@@ -31,26 +31,26 @@ Examples::
     Graph.Arista.Zerotouch.EOS
 
 
-Overlays
+Microkernel docker image
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Overlay Names**
+**Image Names**
 
-We tend to prefix overlays with *overlayfs_*  along with some information about which
-kernel/base image the overlay was built off and information about what is contained
-within the overlay. Overlays are suffixed with *.cpio.gz* because they are gzipped
-cpio archives.
+We tend to prefix docker images with *micro_*  along with some information about which
+RancherOS the docker image was built off and information about what is contained
+within the docker image. Images are suffixed with *docker.tar.xz* because they are xzed
+tar archives contain docker image.
 
 Examples::
 
-    overlayfs_3.13.0-32_flashupdt.cpio.gz
-    overlayfs_3.13.0-32_brocade.cpio.gz
-    overlayfs_3.13.0-32_all_binaries.cpio.gz
+    micro_1.2.0_flashupdt.docker.tar.xz
+    micro_1.2.0_brocade.docker.tar.xz
+    micro_1.2.0_all_binaries.docker.tar.xz
```

Comment:
This is not the case till now.

Reply:
This is a name convention that is better for user to have. It is not required for `on-imagebuilder`.

Reply:
If even we do not use it, why should we propose it?